run_imi.sh

```diff
@@ -197,7 +198,8 @@ cd $InversionPath
 cp $ConfigFile "${RunDirs}/config_${RunName}.yml"
 
 # Upload output to S3 if specified
-cd $InversionPath
-python src/utilities/s3_upload.py $ConfigFile
+if "$S3Upload"; then
+    python src/utilities/s3_upload.py $ConfigFile
+fi
```

Comment:
OK. I can remove it. I originally added it because I was using an older IMI python environment that didn't have boto3 and that caused error messages when calling s3_upload.py (which I didn't need when running from a local cluster anyway). I resolved the error messages by later updating my conda environment.

Reply:
See https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/9629c549521385b0709e1aa33c1c158082f6ddc7